### PR TITLE
Added docstring to rrulewrapper class

### PR DESCRIPTION
**Pull request by sveneschlbeck**
#21880 @StefRe

### CODE REVIEW

**Comment by StefRe** [comment]
hm, the docstring alone doesn't appear to have any effect - see [here](https://65597-1385122-gh.circle-artifacts.com/0/doc/build/html/api/dates_api.html) and [here](https://65597-1385122-gh.circle-artifacts.com/0/doc/build/html/gallery/ticks/date_formatters_locators.html).

File: lib/matplotlib/dates.py [hunk 1]
```diff
@@ -980,6 +980,11 @@ def __call__(self, x, pos=None):
 
 
 class rrulewrapper:
+    """
+    A simple wrapper around a ``dateutil.rrule`` allowing almost
+    arbitrary date tick specifications.
```
Comment (jklymak) [inline lib/matplotlib/dates.py:985]: Maybe "allowing flexible date tick ...".

**Review by dstansby — CHANGES_REQUESTED** [review]
Thanks for the PR! It looks like it's failing our flake8 tests. Could you fix this? It looks like there just a blank line that needs deleting.

**Comment by sveneschlbeck** [comment]
@dstansby Thanks for the hint!

**Review by dstansby — APPROVED** [review]

**Review by timhoffm — APPROVED** [review]

**Pull request merged**